experimental/basisLieHighestWeight/MonomialOrder.jl

```diff
@@ -0,0 +1,110 @@
+# manages methods for different orders of monomials
```

Comment:
OSCAR already has a system of monomial orderings, any reason you are not using that?

Reply:
I will switch to the Julia-Type, but both an older version of the code and parts of the code that create monomials through integer points of a polytope made it easier for me to keep the integers.